Library/Homebrew/dev-cmd/audit.rb

```diff
@@ -597,7 +597,7 @@ def audit_specs
         end
       end
 
-      if formula.head || formula.devel
+      if @official_tap && (formula.head || formula.devel)
```

Comment:
Mind using `formula.core_formula?` instead?

Reply:
this change was removed